rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :book, :customer
+
+  def initialize(date, book, _customer)
```

Comment:
- [ ] The constructor of Rental should be modified so Person is set in it to break the many-to-many relationship, kindly change `_customer` to `person`.